scanners/kube-hunter/parser/parser.js

```diff
@@ -36,4 +36,8 @@ async function parse({ vulnerabilities = [], nodes = [] }) {
   });
 }
 
+function to_valid_url(location){
```

Comment:
Snake case is usually not used in JS.

```suggestion
function toValidUrl(location){
```